### DISCUSSION TITLE
高倍速播放视频经常缓冲

### DISCUSSION THREAD

**Discussion by hhyygg2009** [the1812/Bilibili-Evolved#1809] [综合讨论 / General]
高倍速播放视频经常缓冲
2.5或者以上的速度
播放一阵子就会卡缓冲，缓冲完又继续播一小段又要缓冲了

浏览器是Edge 90.0.818.56
脚本版本1.12.8
网络是电信600M光纤有线、显卡是AMD RX560、

**Comment by Coulomb-G** [comment]
网速不够的话,我只能建议你下载到本地播放

**Reply by hhyygg2009** [reply]
浏览器是Edge 90.0.818.56
脚本版本1.12.8
网络是电信600M光纤有线、显卡是AMD RX560、
电脑配置和网络都没问题的，倍速播视频不应该卡吧

视频就是缓冲了一小段，播到快结束又卡住缓冲

**Comment by u2shana** [comment]
这几个月B站都这样，我觉得是B站CDN问题

**Comment by cxw620** [comment]
B站的CDN调度应该是优先保障客户端的, 网页端可能不怎么看重, 分配的很多都是mCDN或者PCDN这种辣鸡CDN.
实测华为和腾讯云的CDN地址是最佳的, 就是不知道能不能强制走这两个
